src/containers/EditEventPage.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component, Fragment } from 'react'
+import { fetchEventInfo } from '../actions/getEventInfoAction'
+import { fetchActiveProjects } from '../actions/fetchActiveProjectsAction'
+import { setLastLocation } from '../actions/setLastLocationAction'
+import { Header } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import CustomRingLoader from '../components/CustomRingLoader'
+import EventForm from '../components/EventForm'
+import { updateEvent } from '../actions/updateEventAction'
+import moment from 'moment'
+import momentTZ from 'moment-timezone'
+
+class EditEventPage extends Component {
+  state = { event: null }
+
+  componentDidMount () {
+    const path = this.props.location.pathname
+    this.props.setLastLocation(path)
+    if (!this.props.cookies.get(process.env.SESSION || 'WebsiteOne_session') && !this.props.loggedInUser.data) {
+      this.props.history.push({ pathname: '/login' })
+    }
+    if (!this.props.projects.length) {
+      this.props.fetchActiveProjects()
+      this.props.fetchEventInfo(this.props.match.params.slug)
```

Comment:
it seems a bit strange to have a `fetchEventInfo` nested inside an if block checking about the length of the projects array, doesn't it? I think I left it there in our pairing session?